README.md

```diff
@@ -466,6 +466,50 @@ The `book/examples/` directory contains example pallets extending this framework
 * [Polkadot sTPS Benchmarks](https://github.com/paritytech/polkadot-stps)
 * [Zombienet Testing Framework](https://github.com/paritytech/zombienet)
 
+## Development
+
+### Pre-commit Hooks
```

Comment:
Should we mention pre-commit hooks in readme?

Reply:
no, will remove i agree